envs/s3-spark-delta-sharing/docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+version: '3'
+
+services:
+  airflow:
+    image: docker-whirl-airflow:py-${PYTHON_VERSION}-local
+    command: ["singlemachine"]
+    ports:
+      - '5000:5000'  # HTTP (Airflow Web UI)
+    env_file:
+      - .whirl.env
+    volumes:
+      - ${DAG_FOLDER}:/opt/airflow/dags/$PROJECTNAME
+      - ${ENVIRONMENT_FOLDER}/whirl.setup.d:${WHIRL_SETUP_FOLDER}/env.d/
+      - ${DAG_FOLDER}/whirl.setup.d:${WHIRL_SETUP_FOLDER}/dag.d/
+      - ${MOCK_DATA_FOLDER}:/mock-data
+    depends_on:
+      - s3server
+      - sparkmaster
+    links:
+      - s3server:${DEMO_BUCKET}.s3server
+
+
+  s3server:
+    image: localstack/localstack-full:0.12.15
+    ports:
+      - "4563-4584:4563-4584"
+      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
+    environment:
+      - SERVICES=s3
+      - EDGE_PORT=4563
+      - DATA_DIR=/tmp/localstack/data
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - DEMO_BUCKET
+      - AWS_SERVER
+      - AWS_PORT
+      - PORT_WEB_UI
+      - DEBUG=true
+      - S3_SKIP_SIGNATURE_VALIDATION=0
+    volumes:
+      - ./aws.setup.d:/docker-entrypoint-initaws.d
+      - ./.s3-mount:/tmp/localstack
+
+  delta:
+    # image: delta-sharing-server:0.2.0-SNAPSHOT
+    image: krisgeus/delta-sharing:0.2.0-SNAPSHOT
```

Comment:
This docker file is the result of a sbt docker:package command in the delta-sharing git project. To prevent people to check out this project I created thsi personal docker. I hope the docker build will be published form databricks itself soon

Reply:
Added some comment to th edocker compose to explain it